packages/file/src/types/systemd/timesyncd.coffee.md

```diff
@@ -0,0 +1,101 @@
+
+# `timesyncd.coffee.md`
+
+## Example
+
+Overwrite `/usr/lib/systemd/timesyncd.conf.d/10_timesyncd.conf` in `/mnt` to
+set a list of NTP servers by using an array and a single fallback server by
+using a string.
+
+```javascript
+require("nikita").file.types.systemd.timesyncd({
+  target: "/usr/lib/systemd/timesyncd.conf.d/10_timesyncd.conf",
+  rootdir: "/mnt",
+  content:
+    NTP: ["ntp.domain.com", "ntp.domain2.com", "ntp.domain3.com"]
+    FallbackNTP: "fallback.domain.com"
+})
+```
+
+Write to the default target file (`/etc/systemd/timesyncd.conf`). Set a single
+NTP server using a string and also modify the value of RootDistanceMaxSec.
+Note: with `merge` set to true, this wont overwrite the target file, only
+specified values will be updated.
+
+```javascript
+require("nikita").file.types.systemd.timesyncd({
+  content:
+    NTP: "0.arch.pool.ntp.org"
+    RootDistanceMaxSec: 5
+  merge: true
+})
+```
+
+## Schema
+
+    schema =
+      type: 'object'
+      properties:
+        'rootdir':
+          type: 'string'
+          description: """
+          Path to the mount point corresponding to the root directory, optional.
+          """
+        'reload':
+          type: 'boolean', default: null
+          description: """
+          Defaults to true. If set to true the following command will be
+          executed `systemctl daemon-reload && systemctl restart
+          systemd-timesyncd` after having wrote the configuration file.
+          """
+        'target':
+          type: 'string', default: '/etc/systemd/timesyncd.conf'
+          description: """
+          File to write.
+          """
+
+This action uses `file.ini` internally, therefore it honors all
+arguments it provides. `backup` is true by default and `separator` is
+overridden by "=".
```

Comment:
`backup` property in `@file.ini` is one of the type: `string` or `function`. So, we cannot pass `true` as a value. What value can we pass instead?